platforms/nuttx/nuttx-configs/Make.defs.in

```diff
@@ -64,7 +64,12 @@ ifeq ($(WINTOOL),y)
   MKDEP = $(TOPDIR)/tools/mknulldeps.sh
 endif
 
-CFLAGS = -Os -g2 ${CMAKE_C_FLAGS} $(ARCHINCLUDES) \
+# enable precise stack overflow tracking
+ifeq ($(CONFIG_ARMV7M_STACKCHECK),y)
+  INSTRUMENTATIONDEFINES = -finstrument-functions -ffixed-r10
+endif
+
+CFLAGS = -Os -g2 ${CMAKE_C_FLAGS} $(ARCHINCLUDES) $(INSTRUMENTATIONDEFINES) \
```

Comment:
Add to CXXFLAGS as well (below)?

Reply:
@dagar Force Pushed. Are the CXX flags correct and representative of what we use to have? I thought there were common wanings used by both? So it was CXX = CXX + C